x-pack/docs/en/security/operator-privileges/operator-only-functionality.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[operator-only-functionality]]
+=== Operator-only functionality
+There are two broad categories of operator-only functionalities: APIs and dynamic cluster Settings.
```

Comment:
Are operator-only functionalities different from operator privileges? I see the terms are both used in the documentation, but seem the same to me. Would it be worth to clarify?

Reply:
The phrase "operator-only fuctionalities" here represents the things that are only actionable/updatable by an operator, i.e. the things protected by operator privileges. I struggled to come up with better terms. "operator-only resources"? "operator-only actions"? "operator-privileges protected stuff"? I am open to suggestions. Thanks!

Reply:
So if I get it correctly, operator privileges is the feature name, while operator-only functionalities refer to list items. I'm not against the term "functionalities", I was just wondering if the term should have been "privileges" instead.

Reply:
I added some more words in the beginning of this section. Hopefully it makes things clear.